ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/client/Web3JExecutionEngineClient.java

```diff
@@ -13,9 +13,11 @@
 
 package tech.pegasys.teku.ethereum.executionlayer.client;
 
+import static tech.pegasys.teku.spec.config.Constants.EL_QUERY_TIMEOUT;
+import static tech.pegasys.teku.spec.config.Constants.EL_REQUEST_TIMEOUT;
```

Comment:
I think this rename is misleading.  The `EXECUTION_TIMEOUT` and `NON_EXECUTION_TIMEOUT` comes from the execution engine spec so we probably want to keep that naming.